modules/ask_va_api/app/lib/ask_va_api/inquiries/payload_builder/inquiry_payload.rb

```diff
@@ -0,0 +1,134 @@
+
+        def initialize(inquiry_params:, user: nil)
+          @inquiry_params = inquiry_params
+          validate_params!
+          @inquiry_details = InquiryDetails.new(inquiry_params).call
+          @user = user
+          @submitter_profile = SubmitterProfile.new(inquiry_params:, user:, inquiry_details:)
+          @veteran_profile = VeteranProfile.new(inquiry_params:, inquiry_details:)
+          @translator = Translator.new
+        end
+
+        def call
+          payload = {
+            AreYouTheDependent: inquiry_details[:inquiry_about].include?('dependent'),
+            AttachmentPresent: attachment_present?,
+            BranchOfService: nil,
+            CaregiverZipCode: nil,
+            ContactMethod: @translator.call(inquiry_params[:contact_preference]),
+            DependantDOB: family_member_field(:date_of_birth),
+            DependantFirstName: family_member_field(:first)
+          }.merge(additional_payload_fields)
+
+          payload[:LevelOfAuthentication] = 'Unauthenticated' if user.nil?
+
+          payload
+        end
+
+        private
+
+        def additional_payload_fields
+          {
+            DependantLastName: family_member_field(:last),
+            DependantMiddleName: family_member_field(:middle),
+            DependantRelationship: translate_field(:dependent_relationship),
+            InquiryAbout: translate_field(:inquiry_about),
+            InquiryCategory: inquiry_params[:category_id],
+            InquirySource: '722_310_000',
+            InquirySubtopic: inquiry_params[:subtopic_id],
+            InquirySummary: inquiry_params[:subject],
+            InquiryTopic: inquiry_params[:topic_id],
+            InquiryType: nil,
+            IsVeteranDeceased: inquiry_params[:is_veteran_deceased]
+          }.merge(school_state_and_profile_data)
+        end
+
+        def school_state_and_profile_data
+          {
+            LevelOfAuthentication: translate_field(:level_of_authentication),
+            MedicalCenter: inquiry_params[:your_health_facility],
+            SchoolObj: build_school_object,
+            SubmitterQuestion: inquiry_params[:question],
+            SubmitterStateOfSchool: build_state_data(:school_obj, :state_abbreviation),
+            SubmitterStateProperty: build_state_data(:address, :state),
+            SubmitterStateOfResidency: build_residency_state_data,
+            SubmitterZipCodeOfResidency: inquiry_params[:postal_code],
+            UntrustedFlag: nil,
+            VeteranRelationship: translate_field(:veteran_relationship),
+            WhoWasTheirCounselor: counselor_info,
+            ListOfAttachments: list_of_attachments,
+            SubmitterProfile: submitter_profile.call,
+            VeteranProfile: veteran_profile.call
+          }
+        end
+
+        def validate_params!
+          raise InquiryPayloadError, 'Missing required inquiry parameters' if inquiry_params.nil?
+        end
+
+        def attachment_present?
```

Comment:
Nit: I think just because the name of this method implies that it's checking if a single attachment is present, it might be worth considering renaming to `attachments_any?` or because it's very concise removing the method and doing this:

`AttachmentPresent: !list_of_attachments.empty?`